com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -512,11 +512,12 @@ private static Options getCommandLineOptions() {
 
         addOption(options, null, "use-vanilla-lua", false, "DEPRECATED! Use --use-uncompressed-lua-source instead.", true);
         addOption(options, null, "use-uncompressed-lua-source", false, "Use uncompressed and unencrypted Lua source code instead of byte code", true);
+        addOption(options, null, "use-lua-bytecode-delta", false, "Use byte code delta compression when building for multiple architectures", true);
         addOption(options, null, "archive-resource-padding", true, "The alignment of the resources in the game archive. Default is 4", true);
 
         addOption(options, "l", "liveupdate", true, "Yes if liveupdate content should be published", true);
 
-        addOption(options, "ar", "architectures", true, "Comma separated list of architectures to include for the platform", false);
+        addOption(options, "ar", "architectures", true, "Comma separated list of architectures to include for the platform", true);
```

Comment:
Unrelated change, but the "architectures" option should be included when calculating the resource cache key